tasks/nc_apps.yml

```diff
@@ -2,59 +2,46 @@
 - name: "[App] - Parse the item values"
   ansible.builtin.set_fact:
     nc_app_name: "{{ item.key }}"
+    nc_app_source: "{{ item.value.source if item.value.source is defined else item.value }}"
     nc_app_cfg: "{{ item.value }}"
 
-- name: "Install and enable Nextcloud Apps"
-  when: nc_app_cfg is string
+- name: "[App] - Download, install, enable and if provided configure Nextcloud Apps"
```

Comment:
This feature will require example in README and needs to be tested in molecule on any app.

Reply:
Example and test will require at least two apps to test `\"` syntax in dynamically assigned tasks names.

Reply:
Yes. The changes in nc_apps.yml requires an update of the Readme. I'll do so once this Pull Request is commited.
The Readme need a further update as there is now an additional requirement to install geerlinggy.php_version from galaxy.

Reply:
nc_app_source is empty or the download-link. If empty, it is tried to download from official store.